src/shared/cloudformation/activation.ts

```diff
@@ -10,7 +10,8 @@ import { localize } from '../utilities/vsCodeUtils'
 import { CloudFormationTemplateRegistry } from './templateRegistry'
 import { CloudFormationTemplateRegistryManager } from './templateRegistryManager'
 
-export const TEMPLATE_FILE_GLOB_PATTERN = '**/template.{yaml,yml}'
+// ** VSCode glob doesn't expand the first **, so it also needs a second **
```

Comment:
Nit to end all nits: remove the leading `**` because it makes the other `**`s in the comment less meaningful.